.github/workflows/ci.yaml

```diff
@@ -77,7 +77,7 @@ jobs:
             ${{ runner.os }}-build-${{ env.cache-name }}-
             ${{ runner.os }}-build-
             ${{ runner.os }}-
-      - run: cargo clippy --all-targets --all-features -- -D warnings
+      - run: cargo clippy --workspace --all-targets --all-features -- -D warnings
```

Comment:
Should we change the other usages of all to workspace too? E.g. for fmt?

Reply:
`cargo build` supports `--workspace` (and marks `--all` as deprecated too), but `cargo fmt` doesn’t yet. Weird.